### DISCUSSION TITLE
error while adding school email

### DISCUSSION THREAD

**Discussion by Preksha0820** [community/community#139952] [GitHub Education]
### Select Topic Area

Bug

### Body

error adding 2320403101@stu.manit.ac.in: email is already in use  ..can not continue my applicaion

**Comment by AidenJamesYt** [comment]
Hi! 2 Things.

Firsr, make sure to put this in the [GitHub Education](https://github.com/orgs/community/discussions/categories/github-education?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22GitHub+Education%22) Category in the future. 

Second, you likely already have an account with the email. I would try to access the email, if possible.


If you continue to have issues, I recommend reaching out to GitHub's Support Team.

Here is a link to [Github's Support Website](https://support.github.com/).

Here is a link to open a [Support Ticket](https://support.github.com/request).

If you need further assistance, let me know! :D

**Reply by ebndev** [reply]
@AidenJamesYt thanks for helping to point @Preksha0820 in the right direction!

As mentioned, I've gone ahead and moved this post to our `GitHub Education` category.
